### PR TITLE
fix for issue: Retrorun controls #384

### PR DESCRIPTION
**Pull request by navy1978**

### CODE REVIEW

**Comment by navy1978** [comment]
Please dont merge yet (it needs to be tested)

File: packages/games/tools/retrorun/retrorun.sh [hunk 1]
```diff
@@ -104,6 +116,18 @@ else
 fi
 
 
+# Force left analog stick to DPAD
+# Get configuration from distribution.conf and set to FORCE_ANALOG_STICK
+get_setting "force_left_analog_stick"
+echo ${EES}
+if [ "${EES}" == "auto" ] || [ "${EES}" == "false" ] || [ "${EES}" == "none" ] || [ "${EES}" == "0" ]; then
+	sed -i "/^retrorun_force_left_analog_stick/d" ${RRCONF}
+	echo 'retrorun_force_left_analog_stick = auto' >> ${RRCONF}
```
Comment (dhwz) [inline packages/games/tools/retrorun/retrorun.sh:125]: this doesn't seem to be right it should be true or false depending on what is the default setting

Comment (navy1978) [reply packages/games/tools/retrorun/retrorun.sh]: Should be ok now, but I see the beta stuck displaying our logo at startup...

**Comment by navy1978** [comment]
It seems fine. we can merge this

**Review by pkegg — APPROVED** [review]

**Review by dhwz — APPROVED** [review]

**Pull request merged**